lib/generators/module_component/templates/app/services/configuration.rb.erb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module <%= class_name %>
+  class Configuration < Common::Client::Configuration::REST
```

Comment:
Should any of the methods in this class be memoized?

Reply:
It looks like memoization is used consistently across the application for the connection method. I'll go ahead and add it to that method to keep patterns consistent.